src/components/CreateArticlesForm.jsx

```diff
@@ -13,61 +13,58 @@ const CreateArticlesForm = () => {
   };
 
   const handleCategoryChange = (value) => {
-   setSelectedCategory(value);
-   return selectedCategory
+    setSelectedCategory(value);
   };
 
   const onSubmit = async (e) => {
     e.preventDefault();
-    let { title, lead, content, encodedImage } = e.target;
+    let { title, lead, content, encodedImage, premium} = e.target;
```

Comment:
are you getting encodedImage from the event target?

Reply:
my bad, I should have declared outside.